src/handlers/cohort-member.handler.ts

```
@@ -7,6 +7,59 @@ export class CohortMemberHandler {
   constructor(private readonly dbService: DatabaseService) {}
 
+  async handleCohortMemberCreated(data: any) {
+    try {
+      this.logger.debug(
+        `Incoming cohort member created event: ${JSON.stringify(data)}`,
+      );
+
+      const cohortMembershipId: string | undefined = data?.cohortMembershipId;
+      const userId: string | undefined = data?.userId || data?.UserID;
+      const cohortId: string | undefined = data?.cohortId || data?.CohortID;
+      const status: string | undefined = data?.status || data?.MemberStatus || 'active';
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)